src/components/map/MapPage.js

```diff
@@ -107,6 +101,7 @@ const MapPage = () => {
   }, [view.zoom])
 
   const onViewChange = ({ center, zoom, bounds }) => {
+    console.log('onViewChange called', { center, zoom, bounds })
```

Comment:
Delete this `console.log`

Reply:
Thanks. I'm going to leave this one in for now to potentially help the devs debug with the map, because there's a really weird issue going on where the library we're using doesn't trigger `onViewChange` if the map isn't properly focused.